savant/deepstream/drawfunc.py

```diff
@@ -185,4 +194,14 @@ def _draw_central_dot(self, obj_meta: ObjectMeta, artist: Artist, spec: DotDraw)
         )
 
     def _blur(self, obj_meta: ObjectMeta, artist: Artist):
-        artist.blur(obj_meta.bbox)
+        try:
+            artist.blur(obj_meta.bbox)
+        except Exception as exc:
+            self.logger.warning(
+                'Got "%s: %s" when trying to blur the object %s.',
+                type(exc).__name__,
+                exc,
+                obj_meta.bbox,
+                exc_info=exc,
+            )
+
```

Comment:
Run `make reformat` to remove empty line.

Reply:
ok